packages/contracts-bedrock/src/libraries/Predeploys.sol

```diff
@@ -113,8 +113,8 @@ library Predeploys {
         if (_addr == EAS) return "EAS";
         if (_addr == GOVERNANCE_TOKEN) return "GovernanceToken";
         if (_addr == LEGACY_ERC20_ETH) return "LegacyERC20ETH";
-        if (_addr == CROSS_L2_INBOX) return "CrossL2Inbox";
```

Comment:
I don't think we necessarily need to have this extra branch in here, its still possible to add the contracts to genesis without this branch? Just seems more simple to not worry about hard fork logic in here